### ISSUE TITLE
interactive graph network view, when click on one node, show related information

### ISSUE THREAD

**Issue by FuqiX** [FAIRplus/FAIR_wizard#55]
when clicking on a node, showing a resource card, similar to the cards in the non-graphic view?

e.g. when clicking the green PROV button in image 1, show something similar to image 2.

_image 1_
![Screenshot_2021-08-05_13-14-47](https://user-images.githubusercontent.com/54043294/128348203-ab01a25e-fe58-4854-87f1-40e07639635b.png)

_image 2_
![Screenshot_2021-08-05_13-15-00](https://user-images.githubusercontent.com/54043294/128348254-5d4ece3a-6e99-4f8a-be0b-05f7e27acc75.png)

**Comment by theisuru** [comment]
@FuqiX I dont think this is possible with the cytoscape graph library we are using. I will check if its doable.
